gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
```

Comment:
Why are you updating random shit for an MC version update.
Please put this type of stuff in different PRs (Same goes for the indent changes in several files, dont do that)

Reply:
I jusr like the \:/